source/process/mobile-release-process.md

```diff
@@ -9,58 +9,137 @@ Notes:
+### A. (T-minus 12 working days) Cut-off for merging major features
 
-1. PM:
-    - Submit pull request for changelog
+No pull requests for major features should be **merged** to the current release after this date. In special cases, exceptions can be made by the Release Manager.
+
+1. Release Manager:
+    - Post this checklist in Release Checklist channel
+    - Draft Changelog in a WIP PR with updates for highlights, feature additions and bug fixes
+    - Start posting a daily Zero Bug Balance query (posted until zero bugs or day of release)
+2. PM:
+    - Prioritize reviewing major features, ensuring any bugs and UX issues get fixed
+    - Check that all major features are behind a feature flag
+    - Review Jira Backlog and move any tickets that will not be merged to the next release
+3. Dev:
+    - Prioritize reviewing, updating, and merging of pull requests for current release until there are no more tickets in the pull request queue marked for the current release
+      - After the cut-off, any PRs that include significant code changes, require approval of the release manager before merging
+4. Marketing:
+    - Prepare a list of highlights to be included in the next platform release announcement
+
+### B. (T-minus 11 working days) Major feature testing
+
+1. QA:
+    - Prioritize testing merged PRs and resolved tickets
+    - Write and update tests in the Release Testing spreadsheet and in Selenium IDE
+    - Run Selenium IDE tests for updated areas, note Pass/Fail and date tested in the Release Testing spreadsheet
+    
+### C. (T-minus 10 working days) Judgment Day
+
+Day when PM decides which major features are included in the release, and which are postponed.
+
+1. **(Team) Triage Meeting**:
+    - Begin daily triage of tickets
+2. Release Manager:
+    - Post this checklist in Release Checklist channel
+    - Verify all items in the last posted release checklist are complete
+    - Update Changelog PR based on what's in/out of the release
+    - Create meta issue for release in GitHub
+    - Post a link to Native Mobile Apps channel for query of remaining bugs in this release
+3. PM:
+    - Review the JIRA tickets remaining in the current release fix version and push those that won't make it to the next fix version
+
+### D. (T-minus 8 working days) Code Complete and Release Candidate Cut 
+
+**Stabilization** period begins when all features for release have been committed. During this period, only **bugs** can be committed to the release branch. Non-bug pull requests are tagged for next version. Exceptions can be made by the Release Manager during triage.
+
+1. Release Manager:
+    - Post this checklist in Release Checklist channel
+    - Verify all items in the last posted release checklist are complete
+    - Update the GitHub meta issue to include a link to the changelog on the documentation branch
+2. PM:
     - Check the minimum server version required and submit pull request to update in fastlane files
```

Comment:
Is this for PM or dev?

Reply:
@esethna It is a PM task in the original doc - I'll move it to Dev task.